tests/item.spec.js

```diff
@@ -54,6 +58,14 @@ describe('Item', function () {
       expect(price).is.a('number')
       done()
     })
+
+    it('should set netUnitPrice or grossUnitPrice to 0', function (done) {
```

Comment:
Thanks for your effort but it test only the constructor assignment. The validation logic in the `_generateXML`.